### PR TITLE
Add uuid tests

### PR DESCRIPTION
**Pull request by ArtDu**
Tests to verify correct work of converter `java UUID <-> tarantool uuid`

I didn't forget about

- [x] Tests
- [x] Changelog (It's no needed)
- [x] Documentation (It's no needed)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

### CODE REVIEW

File: src/test/resources/org/testcontainers/containers/server.lua [hunk 1]
```diff
@@ -67,6 +67,21 @@ m:create_index('primary', {
     parts = {'id', 'name'}
 })
 
+if _TARANTOOL > "2.4.1" then
```
Comment (ArtDu) [inline src/test/resources/org/testcontainers/containers/server.lua:70]: Doesn't work with "2.10"

Comment (ArtDu) [reply src/test/resources/org/testcontainers/containers/server.lua]: Fixed

File: src/test/java/io/tarantool/driver/TarantoolUtils.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package io.tarantool.driver;
+
+public final class TarantoolUtils {
+    private TarantoolUtils() {
+    }
+
+    public static boolean versionGreaterThen(String version) {
+        return java.lang.System.getenv("TARANTOOL_VERSION") == null ||
+                java.lang.System.getenv("TARANTOOL_VERSION").compareTo(version) > 0;
```
Comment (wey1and) [inline src/test/java/io/tarantool/driver/TarantoolUtils.java:9]: Possible NPE

**Review by wey1and — CHANGES_REQUESTED** [review]

File: src/test/java/io/tarantool/driver/TarantoolUtils.java (hunk 2 above)
Comment (ArtDu) [reply src/test/java/io/tarantool/driver/TarantoolUtils.java:9]: Where exactly?

Comment (ArtDu) [reply src/test/java/io/tarantool/driver/TarantoolUtils.java:9]: Where exactly? We check that` java.lang.System.getenv("TARANTOOL_VERSION")` may be null in first condition

Comment (ArtDu) [reply src/test/java/io/tarantool/driver/TarantoolUtils.java]: Updated, please check new version

**Pull request merged**